src/commands/service/create.ts

```diff
@@ -41,15 +41,23 @@ export interface CommandOptions {
   packagesDir: string;
   pathPrefix: string;
   pathPrefixMajorVersion: string;
+  serviceBuildVg: string;
+  serviceBuildVariables: string;
 }
 
 export interface CommandValues extends CommandOptions {
   k8sPort: number;
   middlewaresArray: string[];
   ringNames: string[];
   variableGroups: string[];
+  serviceVgArray: string[];
+  serviceVariablesArray: string[];
 }
 
+export const validUpperUnderscoreCase = (segment: string): boolean => {
+  return !!segment.match(/^[A-Z0-9_]+$/);
```

Comment:
@yradsmikham we should test this 
<img width="674" alt="image" src="https://user-images.githubusercontent.com/3119513/79494846-889ee680-7fd8-11ea-8db5-311f117df334.png">